src/sshd/install.sh

```diff
@@ -44,7 +44,7 @@ apt_get_update_if_needed()
 {
     if [ ! -d "/var/lib/apt/lists" ] || [ "$(ls /var/lib/apt/lists/ | wc -l)" = "0" ]; then
         echo "Running apt-get update..."
-        apt-get update
+        apt-get update -y
```

Comment:
Instead of running the check twice (initial one and here), can we just remove this function entirely and replace `apt_get_update_if_needed` to be `apt-get update -y` on line 56? That would just remove the conditional part of the update and always run it once.

Do you know the reason why we previously had this behind a conditional? I want to make sure we aren't breaking anything by always running this. Perhaps it was just meant to be a performance optimization, in which case, I think it's fine to always run to prefer correctness over performance.

Reply:
Another reason to move all of this to line 56 is to skip the update if we aren't going to install any packages. The root cause of this bug seems to be that we were skipping the update on line 56 and then trying to install packages on line 57. But we only want to run the update and install if we don't have those packages installed already. So we shouldn't run `apt update` until we know we need to install the missing packages.

Reply:
The cached apt lists are what tell apt what to install, so unless we update them first, apt is not going to know whether or not the packages need to be updated.  

I suppose this could check if _any_ version of `openssh-server openssh-client lsof`, but I think it is just easier to let apt figure that out for us and install the latest compatible versions of each